server/src/main/java/org/candlepin/resource/OwnerProductResource.java

```diff
@@ -182,15 +190,17 @@ protected Content fetchContent(Owner owner, String contentId) {
     @ApiOperation(notes = "Retrieves a list of Products", value = "list")
```

Comment:
Because you are using Response, you need to tell  Swagger which entity will be returned. Use response and responseContainer parameters for this:

```java 
 @ApiOperation(response=Product.class,
        responseContainer="List",
    ...
```